File: src/test/java/com/lsd/CapturedScenarioTest.java
package com.lsd;

import org.junit.jupiter.api.Test;

import static org.assertj.core.api.Assertions.assertThat;

class CapturedScenarioTest {

    private CapturedScenario capturedScenario = new CapturedScenario();

    @Test
    void excludeNullEvents() {
        capturedScenario.add(null);

        assertThat(capturedScenario.getSequenceEvents()).isEmpty();
    }
}

File: src/test/kotlin/com/lsd/CapturedScenarioTest.kt
package com.lsd

import com.lsd.builders.CapturedScenarioBuilder
import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.Test

internal class CapturedScenarioTest {
    private val capturedScenario = CapturedScenarioBuilder().build()

    @Test
    fun excludeNullEvents() {
        capturedScenario.add(null)
        assertThat(capturedScenario.sequenceEvents).isEmpty()
    }
}